src/server/pkg/cmdutil/cobra.go

```diff
@@ -292,6 +294,47 @@ func ParseHistory(history string) (int64, error) {
 	return int64(result), err
 }
 
+// ParseJobFilters parses filter strings into JobFilters
+// Currently each string must be the name of a job state
+func ParseJobFilters(filterStrs []string) (string, error) {
+	var conditions []string
+	for _, stateStr := range filterStrs {
+		var state pps.JobState
+		if value, ok := pps.JobState_value[stateStr]; ok {
+			state = pps.JobState(value)
+		} else {
+			switch strings.ToLower(stateStr) {
+			case "starting":
+				state = pps.JobState_JOB_STARTING
+			case "running":
+				state = pps.JobState_JOB_RUNNING
+			case "merging":
+				state = pps.JobState_JOB_MERGING
+			case "failure":
+				state = pps.JobState_JOB_FAILURE
+			case "success":
+				state = pps.JobState_JOB_SUCCESS
+			case "killed":
+				state = pps.JobState_JOB_KILLED
+			default:
+				return "", fmt.Errorf("states must be one of: starting, running, merging, failure, success, killed")
```

Comment:
This logic right here should probably be relocated `src/client/pps/util.go` it'll probably be useful somewhere else.

Reply:
done